pkg/logging/kafka/create.go

```diff
@@ -124,6 +136,35 @@ func (c *CreateCommand) createInput() (*fastly.CreateKafkaInput, error) {
 		input.Placement = fastly.String(c.Placement.Value)
 	}
 
+	if c.ParseLogKeyvals.Valid {
+		input.ParseLogKeyvals = fastly.CBool(c.ParseLogKeyvals.Value)
+	}
+
+	if c.RequestMaxBytes.Valid {
+		input.RequestMaxBytes = fastly.Uint(c.RequestMaxBytes.Value)
+	}
+
+	if c.UseSASL.Valid && c.UseSASL.Value {
```

Comment:
Move this error conditional towards the top of the function to fail early.

Reply:
Done